tokens/tokens-eth.json

```diff
@@ -10679,6 +10679,30 @@
 			"youtube": "https://www.youtube.com/channel/UCwYD8VSclMxBOAksP5MZJow"
 		}
 	},
+	{
+		"symbol": "PTC",
+		"address": "0xF4980D6a8c038Ec94d12da9446F15E57601e8A5a",
```

Comment:
Is this the correct address? Looking at etherscan, I found this ParrotCoin token https://etherscan.io/address/0x2a8e98e256f32259b5e5cb55dd63c8e891950666

Reply:
Yes. it should be the contract address. not the owner's. Can you please update the PR?

Reply:
i Have updated the code https://github.com/ParrotCoin/ethereum-lists/pull/1